packages/tables/resources/views/components/summary/row.blade.php

```diff
@@ -36,7 +44,9 @@
         <x-filament-tables::cell
             class="text-sm font-medium text-gray-950 dark:text-white"
         >
-            {{ $heading }}
+            <span class="grid gap-y-1 px-3 py-4">
```

Comment:
Are all of these classes really needed?

Reply:
without the classes 
![image](https://github.com/filamentphp/filament/assets/67364036/b26447c6-0999-40e8-b619-e6c50c0e4581)

I'm just copying the styling from text-column, else it can be just a `px-3` or `ps-3`

Reply:
Alright, we don't need the grid and gap classes though.